tests/www/views/test_views.py

```diff
@@ -62,6 +62,21 @@ def test_configuration_expose_config(admin_client):
     check_content_in_response(["Airflow Configuration"], resp)
 
 
+@mock.patch.dict(os.environ, {"AIRFLOW__WEBSERVER__CONFIG_FILE": "/tmp/webserver_config.py"})
+def test_webserver_configuration_config_file(admin_client):
+    conf = initialize_config()
+    conf.validate()
+
+    assert WEBSERVER_CONFIG == os.path.join(get_airflow_home(), "webserver_config.py")
+
+    config_path = os.path.join("/tmp", "webserver_config.py")
+
+    assert os.path.isfile(config_path)
```

Comment:
```suggestion
    config_path = "/tmp/webserver_config.py"
    assert WEBSERVER_CONFIG == config_path)
    assert os.path.isfile(config_path)
```

Let's not use `get_airflow_home`, but otherwise feel free to take or leave these suggestions.

Reply:
Sure. Used the AIRFLOW_HOME instead